packages/components/src/InputFile/InputFile.test.tsx

```diff
@@ -39,6 +39,16 @@ describe("Post Requests", () => {
     expect(container).toMatchSnapshot();
   });
 
+  it("renders an InputFile with custom accepted MIME types", () => {
+    const { container } = render(
+      <InputFile
+        allowedTypes={["image/png", "image/jpg", "application/pdf"]}
+        getUploadParams={fetchUploadParams}
+      />,
+    );
+    expect(container).toMatchSnapshot();
```

Comment:
I'd prefer if we write an assertion test for this. can just find an input and then `toHaveAttribute` it.